### ISSUE TITLE
Expansions not working

### ISSUE THREAD

**Issue by PirataGu** [PlaceholderAPI/PlaceholderAPI#719]
### Confirmation

- [X] My issue isn't already found on the Issue tracker.
- [X] My issue is about **PlaceholderAPI** and not any expansion or external plugin
- [X] The issue isn't already fixed in a Spigot Release or Development Build.

### Type

API Bug

### What happens?

I downloaded an expansion trough papi ecloud and It downloads but it gives an error when reloading and with that it doesnt work...

### Expected Behaviour

Download the expansion ( which does ) and placeholders start working ( which doesn't ).

### How to Reproduce

1. papi ecloud download <expansion>
2. papi reload
3. error on console and the placeholders doesnt work

### `/papi dump` Output

https://paste.helpch.at/dolonodazu

### Console Log

https://paste.helpch.at/ahadalesom.md

### Errors

https://paste.helpch.at/tedaqimasi.sql

**Comment by Andre601** [comment]
What expansion are you actually talking about?

Also, your log has a lot (and I mean A LOT) of other errors including one of a Per-World plugin manager, which imo can also be a cause for errors, so you may want to also fix those first...

**Comment by PirataGu** [comment]
> 
> 
> What expansion are you actually talking about?
> 
> Also, your log has a lot (and I mean A LOT) of other errors including one of a Per-World plugin manager, which imo can also be a cause for errors, so you may want to also fix those first...

Yes I know because I'm changing my server to java-16 and to do that I had to change to 1.16.5 which caused a lot of error and Im fixing it ... okay I'm going to fix first perworldplugin

**Comment by Starmism** [comment]
Not a PlaceholderAPI issue.
